gratipay/models/team.py

```diff
@@ -158,19 +158,6 @@ def to_dict(self):
         }
 
     def migrate_tips(self):
-        payment_instructions = self.db.all("""
-            SELECT pi.*
-              FROM payment_instructions pi
-              JOIN teams t ON t.slug = pi.team
-              JOIN participants p ON t.owner = p.username
-             WHERE p.username = %s
-               AND pi.ctime < t.ctime
-        """, (self.owner, ))
-
```

Comment:
@rohitpaulk in case we decide to keep this, I think the `JOIN participants p ON t.owner = p.username WHERE p.username = %s` is unnecessary. A simple `where t.owner = %s` should suffice.